src/test/java/com/exactpro/th2/codec/qfj/QFJCodecTest.java

```diff
@@ -327,7 +332,10 @@ private static void initMessages() {
 
     @BeforeAll
     private static void initQFJCodec() throws ConfigError {
-        codec = new QFJCodec(new QFJCodecSettings(), null, new DataDictionary("src/test/resources/FIXT11.xml"), new DataDictionary("src/test/resources/FIX50SP2.xml"));
+        QFJCodecSettings settings = new QFJCodecSettings();
+        settings.setCheckFieldsOutOfOrder(true);
```

Comment:
Maybe use default settings in general and create separate codec instances to test behaviour with special setting